api/healthCheck/index.ts

```diff
@@ -47,26 +44,27 @@ export function getPromises(path): any[] {
             break
         }
     }
-
-    const health = getConfigValue(HEALTH)
-
     if (healthCheckEndpointDictionary[path]) {
+        console.log(healthCheckEndpointDictionary[path])
         healthCheckEndpointDictionary[path].forEach(endpoint => {
-            Promises.push(http.get(health[endpoint]))
+            // TODO: Have health config for this.
+            console.log('healthEndpoints', endpoint)
+            console.log(healthEndpoints()[endpoint])
```

Comment:
`console.log()` to go, unless we really need it.

Reply:
removed console.log, now using the correct logger.